tests/detect_test_class.py

```diff
@@ -12,14 +12,24 @@
 from lunavl.sdk.image_utils.geometry import Rect
 from lunavl.sdk.image_utils.image import VLImage
 from tests.base import BaseTestClass
-from tests.resources import ONE_FACE, SEVERAL_FACES, SMALL_IMAGE, BAD_IMAGE
+from tests.resources import ONE_FACE, SEVERAL_FACES, SMALL_IMAGE, BAD_IMAGE, LARGE_IMAGE
 
 VLIMAGE_SMALL = VLImage.load(filename=SMALL_IMAGE)
 VLIMAGE_ONE_FACE = VLImage.load(filename=ONE_FACE)
+VLIMAGE_LARGRE = VLImage.load(filename=LARGE_IMAGE)
 BAD_IMAGE = VLImage.load(filename=BAD_IMAGE)
 VLIMAGE_SEVERAL_FACE = VLImage.load(filename=SEVERAL_FACES)
 GOOD_AREA = Rect(100, 100, VLIMAGE_ONE_FACE.rect.width - 100, VLIMAGE_ONE_FACE.rect.height - 100)
 OUTSIDE_AREA = Rect(100, 100, VLIMAGE_ONE_FACE.rect.width, VLIMAGE_ONE_FACE.rect.height)
+SMALL_AREA = Rect(50, 50, 52, 52)
+LARGE_AREA = Rect(1, 1, VLIMAGE_LARGRE.rect.width - 1, VLIMAGE_LARGRE.rect.height - 1)
```

Comment:
Better, intialize this in test with image context

Reply:
This initialization is the same as for the others added earlier. If the old initialization is bad, why is it in the repository?

Reply:
hm. you are right.